File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/ForholdstallUtvalg.java
package no.nav.domain.pensjon.regler.grunnlag;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class ForholdstallUtvalg implements Serializable {


    private double ft;
    private List<Forholdstall> forholdstallListe = new ArrayList<Forholdstall>();


}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/ForholdstallUtvalg.kt
package no.nav.pensjon.regler.domain.grunnlag

import java.io.Serializable

class ForholdstallUtvalg : Serializable {
    var ft = 0.0
    var pREG_ft67soker = 0.0
    var pREG_ft67virk = 0.0
    var pREG_reguleringsfaktor = 0.0
    var forholdstallListe: List<Forholdstall> = mutableListOf()

}